internal/event_handlers.go

```diff
@@ -3,7 +3,7 @@ package internal
 type (
 	MenuButtonClickedEventHandler          func()
 	SubmitButtonClickedEventHandler        func(bool)
-	PullRequestMenuItemClickedEventHandler func()
-	AnalyzeButtonClickedEventHandler       func()
-	OnGetPullRequestEvent                  func(pullRequest string)
+	PullRequestMenuItemClickedEventHandler func(gptModel string)
+	AnalyzeButtonClickedEventHandler       func(gptModel string)
+	OnGetPullRequestEvent                  func(gptModel, pullRequest string)
```

Comment:
This line is probably the correct naming convention.  The other event handler names perhaps should follow the `OnEventName` convention? 

This could be a good time to sneak in that rename.

Reply:
good catch